tmd/view/view.py

```diff
@@ -1115,10 +1113,166 @@ def polar_plot(pop, neurite_type="neurites", bins=20):
     """Generate a polar plot of a neuron or population."""
     input_data = _get_polar_data(pop, neurite_type=neurite_type, bins=bins)
 
-    fig = _cm.plt.figure()
+    fig = plt.figure()
     ax = fig.add_axes([0.1, 0.1, 0.8, 0.8], polar=True)
 
-    theta = _np.array(input_data)[:, 0]
-    radii = _np.array(input_data)[:, 2] / _np.max(input_data)
-    width = 2 * _np.pi / len(input_data)
+    theta = np.array(input_data)[:, 0]
+    radii = np.array(input_data)[:, 2] / np.max(input_data)
+    width = 2 * np.pi / len(input_data)
     ax.bar(theta, radii, width=width, bottom=0.0, alpha=0.8)
+
+
+def tree_colors(
+    tree,
```

Comment:
There is a function called `tree` so I had to add `# pylint: disable=redefined-outer-name`, but maybe we can change the name of this parameter?